config/eloquent_driver.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+return [
+
+    'connection' => env('STATAMIC_ELOQUENT_CONNECTION', ''),
+    'table_prefix' => env('STATAMIC_ELOQUENT_PREFIX', ''),
+
+    'assets' => [
+        'driver' => 'eloquent',
+        'container_model' => \Statamic\Eloquent\Assets\AssetContainerModel::class,
+        'model' => \Statamic\Eloquent\Assets\AssetModel::class,
+    ],
+
+    'blueprints' => [
+        'driver' => 'eloquent',
+        'blueprint_model' => \Statamic\Eloquent\Fields\BlueprintModel::class,
+        'fieldset_model' => \Statamic\Eloquent\Fields\FieldsetModel::class,
+    ],
+
+    'collections' => [
+        'driver' => 'eloquent',
+        'model' => \Statamic\Eloquent\Collections\CollectionModel::class,
+        'tree' => \Statamic\Eloquent\Structures\CollectionTree::class,
+        'tree_model' => \Statamic\Eloquent\Structures\TreeModel::class,
```

Comment:
@FrittenKeeZ The `\Statamic\Eloquent\Structures\TreeModel::class` is now referenced in two places. Once under `collections` and once in `navigations`. Should this not be just define in one place?

Reply:
No, this is to allow you to define a different tree model for each if desired.

Reply:
But is it not the exact same DB table?

Reply:
Yes it is. But the point is that it doesn’t have to be - if someone wants to use a different model or table they can do. It’s giving flexibility.

Reply:
Ah, make sense. Nice!